src/homepageExperience/components/steps/arduino/PrepareIde.tsx

```diff
@@ -0,0 +1,53 @@
+// Libraries
+import React from 'react'
+
+const listStyle = {
+  fontSize: '16px',
+  fontWeight: 'normal',
+}
+
+export const PrepareIde = () => {
+  return (
+    <>
+      <h1>Prepare Arduino IDE</h1>
+      <p>
+        The easiest way to get started with the InfluxDB Arduino client is with
+        the ESP8266 or ESP32 board.
+      </p>
+      <p>
+        If you haven't already, add the board you wish to use (ESP8266 or ESP32)
+        to the Arduino IDE by following these steps:
+      </p>
+      <p>
+        <b>For ESP8266:</b>
+      </p>
+      <ol style={listStyle}>
+        <li>Open the Arduino Preferences (Arduino {'>'} Preferences) </li>
```

Comment:
regarding the use of `>`:

This is going to get kind of deep into an accessbility rabbit hole, but the `>` (greater than) sign is meant for comparing numbers (e.g. 5 > 2). I think a better symbol is the right arrow → which can be rendered by using `&rarr;` in lieu of `{'>'}`. I used [this site](https://www.deque.com/blog/dont-screen-readers-read-whats-screen-part-1-punctuation-typographic-symbols/) as a guide.

Reply:
This is a good point, i didn't think of that when sending david the copy. Thank you!

Reply:
Good point, I fixed that. Thanks for the reference!